src/order/filter/filter_product.lean

```diff
@@ -29,158 +29,155 @@ local notation `β*` := filterprod β φ
 
 namespace filter_product
 
-def seq_to_filterprod : (α → β) → β* := @quotient.mk' (α → β) (bigly_equal β φ)
+variables {α β φ} include α β φ
```

Comment:
The `include` statement ought not be necessary. If some of the code breaks after you remove it, that's an indicator that some variables need to be explicit in that particular statement.

Reply:
I removed the `include`s on `α β`, I don't think I can do anything about `φ`. What do you mean some variables need to be explicit?